CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+Thank you for your interest in contributing to Codewind.
+
+#### Before you contribute
```

Comment:
We have already pointed user to the commit instruction from our codewind github workflow doc.  We can remove this section. The link to here refers to bugzilla and Gerrit etc. that doesn't applies to us.